bincommon/commonflags.go

```diff
@@ -114,8 +100,9 @@ var (
 // It sets up the common flags, the rest of usage/argument/flag handling
 // is now moved to the [fortio.org/cli] and [fortio.org/scli] packages.
 func SharedMain() {
-	flag.Var(&headersFlags, "H",
-		"Additional HTTP header(s) or gRPC metadata. Multiple `key:value` pairs can be passed using multiple -H.")
+	flag.Func("H",
+		"Additional HTTP header(s) or gRPC metadata. Multiple `key:value` pairs can be passed using multiple -H.",
+		func(s string) error { return httpOpts.AddAndValidateExtraHeader(s) })
```

Comment:
what the linter says, the extra func is not doing anything

Reply:
fix it